jjb/dynamic/scale-ci_mastervertical.yml

```diff
@@ -0,0 +1,169 @@
+        # Re-enable logging
+        set -x
+
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        export PBENCH_SSH_PUBLIC_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PBENCH_SSH_PRIVATE_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+
+        # Create inventory File:
+        echo "[orchestration]" > inventory
+        echo "${ORCHESTRATION_HOST}" >> inventory
+
+        export ANSIBLE_FORCE_COLOR=true
+        ansible --version
+        time ansible-playbook -vv -i inventory workloads/mastervertical.yml
+
+        oc logs --timestamps -n scale-ci-tooling -f job/scale-ci-mastervertical
+
+        oc get job -n scale-ci-tooling scale-ci-mastervertical -o json | jq -e '.status.succeeded==1'
+    concurrent: true
+    description: |
+      MasterVertical workload is used to measure control plane density for OpenShift. This test loads several objects across the cluster using cluster loader.
+      This job is managed by https://github.com/openshift-scale/scale-ci-pipeline
+    disabled: false
+    name: ATS-SCALE-CI-MASTERVERTICAL
+    node: scale-ci
+    parameters:
+    - string:
+        default: "kubeadmin"
+        description: User name to access cluster.
+        name: CLUSTER_USER
+    - string:
+        default: ""
+        description: Password for CLUSTER_USER to access cluster.
+        name: CLUSTER_PASSWORD
+    - string:
+        default: ""
+        description: The URL address to the openshift cluster to login to.
+        name: CLUSTER_API_URL
+    - string:
+        default: ""
+        description: Token to access private repo containing ssh keys.
+        name: SSHKEY_TOKEN
+    - string:
+        default: "localhost"
+        description: The machine intended to run the oc commands and launch the workload.
+        name: ORCHESTRATION_HOST
+    - string:
+        default: "root"
+        description: The user for the Orchestration host.
+        name: ORCHESTRATION_USER
+    - string:
+        default: "quay.io/openshift-scale/scale-ci-workload"
+        description: Container image that runs the workload script.
+        name: WORKLOAD_IMAGE
+    - bool:
+        default: false
+        description: Enables/disables the node selector that places the workload job on the `pbench` node.
+        name: WORKLOAD_JOB_NODE_SELECTOR
+    - bool:
+        default: false
+        description: Enables/disables the toleration on the workload job to permit the `controller` taint.
+        name: WORKLOAD_JOB_TAINT
+    - bool:
+        default: false
+        description: Enables/disables running the workload pod as privileged.
+        name: WORKLOAD_JOB_PRIVILEGED
+    - string:
+        default: "/root/.kube/config"
```

Comment:
I can't recall if there is a particular reason we have `/root/.kube/config` configured here instead of `~/.kube/config`,  should we make sure all the workloads have `~/.kube/config` as the default?